src/lib/components/composites/criteria/CriteriaList.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+    import CriterionListEntry from "./CriterionListEntry.svelte";
+    import type { ReviewedCriterion } from "$lib/model/general";
+    import type { User } from "$lib/model/api/user";
+    import CriterionListEntrySkeleton from "./CriterionListEntrySkeleton.svelte";
+
+    interface Props {
+        listTitle: string;
```

Comment:
```suggestion
        listTitle: "Hard Exclusion" | "Soft Exclusion" | "Inclusion";
```
as we actually want only this three types of criteria lists and it should not be easily possible to add a new list type, as this would require a lot of more work, dont you agree?

Reply:
Good catch. I changed it.